adal/src/main/java/com/microsoft/aad/adal/Discovery.java

```diff
@@ -255,4 +255,8 @@ private URL buildQueryString(final String instance, final String authorizationEn
                 .appendQueryParameter(AUTHORIZATION_ENDPOINT_KEY, authorizationEndpointUrl);
         return new URL(builder.build().toString());
     }
+
+    Set<String> getValidHosts() {
```

Comment:
HttpEvent.setHttpPath, Aashima's spec says we log http path only if its well known.

---
In reply to: [90364666](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90364666) [](ancestors = 90364666)